workflows/multiomics/full_pipeline/integration_test.sh

```diff
@@ -16,4 +16,21 @@ bin/nextflow \
   -entry test_wf \
   -resume \
   -profile docker,no_publish \
-  -c workflows/utils/labels_ci.config
\ No newline at end of file
+  -c workflows/utils/labels_ci.config
+
+bin/nextflow \
```

Comment:
I've been removing `bin/` from scripts whenever I make changes, so we can deprecate the `bin/init` at some point

Reply:
We should tackle this in a separate PR

Reply:
#252